Cargo.toml

```diff
@@ -18,12 +18,19 @@ std = []
 serde-std = ["serde/std"]
 unstable = []  # for benchmarking
 fuzztarget = [] # used by other rust-bitcoin projects to make hashes almost-noops, DON'T USE THIS
+json = ['schemars']
```

Comment:
Hmm this might confuse users that they need this feature to have serde serialization for json. Could we either just use the `schemars` feature without adding an explicit feature for it or use something more specific like `json-schema`?

Reply:
I agree, I don't understand at all what the point of renaming the feature is